### DISCUSSION TITLE
Introducing `fatality`

### DISCUSSION THREAD

**Discussion by winstxnhdw** [winstxnhdw/lc-hax#191] [Announcements]
I've recently added the new `fatality` command. I think this is a feature that was previously highly requested but thought to be server-sided. I haven't tested it myself so I need volunteers.

```console
/fatality <player> <enemy>
```

Currently, I only support fatality animations for `giant`, `jester`, and `mask`.

Enjoy and give me your feedback!

**Comment by yujispin990** [comment]
OH MY WHAT!?!

**Reply by yujispin990** [reply]
I've used something similar to this but i thought it was not possible anymore?! you are simply the best @winstxnhdw 😍

**Reply by yujispin990** [reply]
gonna test it tonight.

**Reply by winstxnhdw** [reply]
no promises that it works though

**Reply by winstxnhdw** [reply]
Hey, I only support 3 enemies right now as stated in this discussion.

For suggestions, you can post them under a new discussion thread.

**Reply by o777l** [reply]
Somehow missed that, I will try again! Thanks for you work on this it's great fun to use

**Comment by winstxnhdw** [comment]
I have implemented all possible fatality animations in latest. Surprisingly, not every mob has a kill animation. Spider, Snare Flea, Hoarding Bug, Coil Head, Leviathan, Spore Lizard and Ghost Girl do not have kill animations.

There's also a fuzzy word matcher so that you do not have to type the full names every time.

**Reply by o777l** [reply]
Oh I thought the kill animations were supposed to be the head popping of from the coil head and ghost girl etc.

You think it's possible to trigger something like that in conjuction with the kill command? If it's not an animation it must be something else right

**Reply by winstxnhdw** [reply]
That's what I thought too but I can't find any code related to popping their heads in Coil Head's or Ghost Girl's
